Sources/TuistCloud/OpenAPI/cloud.yaml

```diff
@@ -0,0 +1,37 @@
+openapi: "3.0.3"
```

Comment:
Who is the source of truth of this one, `tuist` I guess?

Reply:
It's currently loosely coupled with the cloud backend. However, I do think this should stay in `tuist`. `tuist` defines the API specification and then `tuist/cloud` is one backend implementation of that specification.